zebra-rpc/src/methods.rs

```diff
@@ -938,6 +954,44 @@ where
         }
         .boxed()
     }
+
+    fn get_block_hash(&self, index: i32) -> BoxFuture<Result<GetBlockHash>> {
```

Comment:
```suggestion
    /// Response to a `getblockhash` RPC request.
    ///
    /// See the notes for the [`Rpc::get_block_hash` method].
    fn get_block_hash(&self, index: i32) -> BoxFuture<Result<GetBlockHash>> {
```

Reply:
No other function has docs at the top in this implementation. Also, i think the suggestion is wrong as this function is not just the response but the implementation of the rpc method declared as part of the `Rpc` trait.